integration_tests/Cargo.toml

```diff
@@ -1,29 +1,31 @@
 [package]
-name = "integration_tests"
-version = "0.1.0"
 edition = "2021"
+name = "integration_tests"
 publish = false
+version = "0.1.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dev-dependencies]
-api = { path = "../api" }
-anyhow = { workspace = true }
-base64 = { workspace = true }
-borsh = { workspace = true }
-bs58 = { workspace = true }
-dao = { path = "../lib/dao" }
-parser = { path = "../lib/parser" }
-persist = { path = "../lib/persist" }
-lazy_static = { workspace = true }
-light-poseidon = { workspace = true }
-log = { workspace = true }
-thiserror = { workspace = true }
-sea-orm = { workspace = true }
-sqlx = { workspace = true }
-solana-sdk = { workspace = true }
-serde_json = { workspace = true }
-serde = { workspace = true }
-tracing = { workspace = true }
-tracing-subscriber = { workspace = true }
+anyhow = {workspace = true}
```

Comment:
This is VSCode autoformat.

@NicolasPennie what are your VSCode linter settings? I don't really care what linter settings we use here, but I just want to make sure we use the same one so that we don't do unnecessary linting changes.